sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -2456,6 +2459,46 @@ def deinit(self):
 
         del globals()['platform_chassis']
 
+    def load_feature_flags(self):
```

Comment:
why not use a similar solution as the way to control the disable/enable of cmis manager task?
https://github.com/sonic-net/sonic-buildimage/blob/9b08fe4eb2fdff9d9e9ebed640100abb3b9aeef8/dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2#L99

Reply:
The reasons are: 
1.  With this method, user can easily enable/disable this feature at runtime for dev/debug/workaround/etc purposes. Just change the flag in pmon json, and restart xcvrd. 
2.  With this method, now xcvrd directly gets the flag from pmon json file, which is more simplified than the old method. Old method needs to first get the flag from json file, generate the supervisord.conf at build time, pass it as xcvrd process startup argument, and finally xcvrd parse the process startup argument to get the flag.